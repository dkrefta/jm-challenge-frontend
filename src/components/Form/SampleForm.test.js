import React from 'react';
import { shallow } from 'enzyme';
import MaskedInput from './SampleForm.jsx';

describe('Input', () => {
 it('should be defined', () => {
   expect(SampleForm).toBeDefined();
 });
 it('should render correctly', () => {
   const tree = shallow(
     <MaskedInput name='input test' />
   );
   expect(tree).toMatchSnapshot();
 });
});