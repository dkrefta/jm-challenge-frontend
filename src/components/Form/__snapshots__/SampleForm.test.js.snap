// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Input
    name="input test"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "error": undefined,
      "id": "cnpj",
      "label": undefined,
      "mask": undefined,
      "name": "cnpj",
      "onBlur": undefined,
      "onChange": undefined,
      "placeholder": undefined,
      "render": [Function],
      "type": "text",
      "value": undefined,
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "error": undefined,
        "id": "cnpj",
        "label": undefined,
        "mask": undefined,
        "name": "cnpj",
        "onBlur": undefined,
        "onChange": undefined,
        "placeholder": undefined,
        "render": [Function],
        "type": "text",
        "value": undefined,
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
